Fix CommandHandler registration issue identified by DRT failure

ryalanms opened the pull request:
Fixes DRT failures and #1022.

stevenbrix commented:
I looked back over #919 and it's not immediately obvious how this regression started happening, and why these exact changes are the fix. Can you provide more information on what happened? That way we can all learn and maybe be lucky enough to avoid these issues in the future.

ryalanms commented:
Translations in .NET Framework were in different file formats and split across multiple files.  .resx does not support the metadata contained in these files.  .NET Framework contained strings in the satellite assemblies that were not translated for any language that were used to register key commands.  Some of these key commands were translated when we moved to .NET Core causing crashes at runtime due to lookup failures.  The original change moves the static strings out of the satellite assemblies to prevent accidental translation.

vatsan-madhavan commented:
Please hold for review from  @rladuca or me.
